### ISSUE TITLE
Questions about pyramidNET

### ISSUE THREAD

**Issue by Shaw95** [mhariat/Fast-PBA_AutoAugment#1]
Hi, I have a few questions about the training of PyramidNET
1. How long did you take to fully train pyramidNET with horovod (1 node, 4 GPUs?)
2. The version of pytorch and horovod?
3. Any tricks when implementing horovod, because I cannot reproduce the accuracy now

Thank you so much!

**Comment by mhariat** [comment]
Hi, Thank you for your question. 

1. The training took me around 2 or 3 days (1800 epochs) with 4 GPUs (16 GB each).

2. I used Pytorch version 1.3.1 and Horovod version 0.18.1.

3. I didn't use any particular tricks. However, you have to be careful with the learning rate schedule. 
As you may have seen, I used a LR Scheduler. The learning rate starts at 0.5. Then it's decreased by a factor 0.1 at epoch 450, by a factor 0.004 at epochs 900 and by a factor 0.0004 at epochs 1250. The reason why I decreased the learning rate that much at epochs 900 is because I observed overshooting issues for learning rate that were neither "too big" nor "too small" (typically in the order of 10^-2)

What accuracy are you getting? Are you using PBA or Fast-AutoAugment for the data-augmenation ?
Please let me know if you have any other questions.
